drivers/hmis/app/models/hmis/project_config.rb

```diff
@@ -25,6 +25,17 @@ def exactly_one_of_project_org_type
 
   validate :validate_config_options_json
 
+  scope :viewable_by, ->(user) do
+    # Special case this, rather than using ProjectRelated concern, because project isn't a direct relationship.
+    # Project config can apply to a project, an organization, or a project type.
+    # Right now we have no way to gate viewability by data source for ProjectConfigs defined by project type - TODO(#6691)
+    project_ids = Hmis::Hud::Project.viewable_by(user).pluck(:id)
```

Comment:
I suspect there is a better more efficient way to structure this query. My rails chops weren't quite up to the task yet so I'd be glad for pointers!